webapp/app.py

```
@@ -779,5 +779,63 @@ def language(lang_code):
+        image_url = response.data[0].url
+        if not image_url:
+            return "Image generation failed", 500
+
+        # Download and cache the image
+        import urllib.request
+
+        os.makedirs(cache_dir, exist_ok=True)
+        urllib.request.urlretrieve(image_url, cache_path)
+
+        return app.send_static_file(f"word-images/{lang_code}/{word.lower()}.webp")
+    except Exception as e:
```

Ruff (0.15.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)